File: server/tests/fixtures/alpha_rule_triggers/javascript/arch_critical_dep_consumer2.js
// architecture.critical_dependency fixture - dependent 2
// Depends on the critical dependency

import { CriticalDependency, criticalUtility } from './arch_critical_dep_core';

export function consumerTwo() {
    const dep = new CriticalDependency();
    return dep.coreOperation("from_two");
}

File: server/tests/fixtures/alpha_rule_triggers/python/arch_critical_dep_consumer2.py
# architecture.critical_dependency fixture - dependent 2
# Depends on the critical dependency

from arch_critical_dep_core import CriticalDependency, critical_utility

def consumer_two():
    """Second consumer of critical dependency."""
    dep = CriticalDependency()
    return dep.core_operation("from_two")
